modules/ROOT/pages/maven-reference.adoc

```diff
@@ -77,6 +77,73 @@ When you create a new Mule project in Anypoint Studio, the required Mule reposit
 </pluginRepository>
 ----
 
+== MuleSoft Enterprise Repository
+
+To add the MuleSoft Enterprise repository to your project, you need to meet the following prerequisites:
+
+* Own an Enterprise license
```

Comment:
I suggest replacing the list with tasks or requirements subtopics:

=== Obtain and Install an Enterprise License

=== Acquire Access to Enterprise Repositories

=== Configure Maven to Access Enterprise Repositories

Reply:
```suggestion
=== Obtain and Install an Enterprise License
```